src/enum/emits.ts

```diff
@@ -1,4 +1,4 @@
 export const enum EmitEvents {
   CLICKED = 'clicked',
-  RESET = 'reset',
+  UPDATE = 'update:modelValue',
```

Comment:
UPDATE is not a convenient term for 'update:modelValue'. Lets call it more verbose, like UPDATE_MODEL_VALUE